templates/registration/profile.html

```diff
@@ -21,13 +21,16 @@ <h2 class="profile-header">Welcome, {% firstof user.get_full_name user.get_usern
             <div class="break">
                 <hr>
             </div>
-
+                <h5 class="profile-total-header">TOTAL EARNED:</h5>
```

Comment:
This looks better, now this line just needs to be unindented to line up with the `{% if total_earned %}` line.